src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor().toString();
```

Comment:
There is no need to call `toString()` on the result of `getRandomColor()` if the method already returns a `String`. If it returns an enum or another type that has a meaningful `toString()` implementation, it is okay to use it. Otherwise, consider revising the `ColorSupplier` implementation to return a `String` directly.